### PR TITLE
Add rubygem-azure_mgmt_subscriptions package

### PR DESCRIPTION
**Pull request by apuntamb**
Supported Versions:

 * Nightly
 * 2.0
 * 1.24
 * 1.23

### CODE REVIEW

File: comps/comps-foreman-plugins-rhel7.xml [hunk 1]
```diff
@@ -136,30 +107,60 @@
...
+      <packagereq type="default">tfm-rubygem-smart_proxy_dynflow</packagereq>
       <packagereq type="default">tfm-rubygem-smart_proxy_dynflow_core</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_m2</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_monitoring</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_omaha</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_openscap</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_pulp</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_realm_ad_plugin</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_remote_execution_ssh</packagereq>
+      <packagereq type="default">tfm-rubygem-smart_proxy_salt</packagereq>
```
Comment (apuntamb) [inline comps/comps-foreman-plugins-rhel7.xml:161]: @evgeni Is all of this an expected reordering or do I need to change it to original?

Comment (evgeni) [reply comps/comps-foreman-plugins-rhel7.xml:161]: This looks perfectly reasonable. Seems we did not reorder the packages when we added the tfm prefix for the SCLed proxy.

Comment (apuntamb) [reply comps/comps-foreman-plugins-rhel7.xml:161]: Great! For it to go into `2.1`, do I need to create a cherrypick PR yet?

Comment (evgeni) [reply comps/comps-foreman-plugins-rhel7.xml:161]: As far as I know 2.1 didn't branch yet.

Comment (apuntamb) [reply comps/comps-foreman-plugins-rhel7.xml:161]: Awesome, thanks!

File: rel-eng/tito.props [hunk 2]
```diff
@@ -866,6 +866,7 @@ whitelist = nodejs-babel-plugin-module-resolver
   rubygem-azure_mgmt_network
   rubygem-azure_mgmt_resources
   rubygem-azure_mgmt_storage
+  rubygem-azure_mgmt_subscriptions
```
Comment (ehelms) [inline rel-eng/tito.props:869]: Please add it to the foreman-nightly-el8 whitelist section as well so that this is available on EL8.

Comment (apuntamb) [reply rel-eng/tito.props:869]: Should I just add `rubygem-azure_mgmt_subscriptions` or all the `rubygem-azure_mgmt_*` as I can't see any of those in the `foreman-nightly-el8` section?

Comment (ehelms) [reply rel-eng/tito.props:869]: Ohh sorry, I meant add to the `foreman-plugins-el8` section! I think there you will fin dthe rest of the azure packages.

Comment (apuntamb) [reply rel-eng/tito.props:870]: done.

**Pull request merged**